### PR TITLE
fix: [PLG-2277]: Added AB test for CD Get Started Page 

### PR DESCRIPTION
**Pull request by devinder-harness**
### Summary
This PR adds an AB test and a new variant for CD Get Started Page
JIRA https://harness.atlassian.net/browse/PLG-2277

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by PratyushGar — APPROVED** [review]

File: src/modules/10-common/pages/welcome/WelcomePageV2.tsx [hunk 1]
```diff
@@ -163,46 +154,13 @@ export default function WelcomePageV2(): JSX.Element {
           },
           disabled: updatingDefaultExperience
         }
-      case 'chaos':
```
Comment (abhinavrastogi-harness) [inline src/modules/10-common/pages/welcome/WelcomePageV2.tsx:166]: why is this PR touching Chaos' get started flow?

File: src/modules/75-cd/strings/strings.en.yaml [hunk 2]
```diff
@@ -814,6 +816,20 @@ getStartedWithCD:
   artifactInManifest: No, it is hard-coded in my manifest
   giveSample: Give me a Sample
   artifactReferencedInManifest: Fantastic! If your artifact is already referenced in your manifest you are ready to continue to the next step.
+  tryCICTA: '<b>Developer</b>? Check out Harness <a target="_blank" href="{{ciurl}}">Continuous Integration</a> to build and test your code x4 faster with hosted builds and test intelligence. <a target="_blank" href="{{docurl}}">CI or CD?</a>'
```
Comment (abhinavrastogi-harness) [inline src/modules/75-cd/strings/strings.en.yaml:819]: x4 or 4x?

**Review by abhinavrastogi-harness — APPROVED** [review]
no tests being added/updated?

**Review by YounixIO — APPROVED** [review]

File: src/modules/10-common/pages/welcome/WelcomePageV2.tsx (hunk 1 above)
Comment (devinder-harness) [reply src/modules/10-common/pages/welcome/WelcomePageV2.tsx:166]: the code removed was for starting trials thats no longer valid now

**Comment by devinder-harness** [comment]
rerun coverage

**Comment by devinder-harness** [comment]
retrigger cypress-pipeline

**Comment by devinder-harness** [comment]
retrigger cypress-cv

**Comment by devinder-harness** [comment]
retrigger coverage

**Pull request merged**